docs/http_api.rst

```diff
@@ -720,6 +763,10 @@ Teams
     Archiving can be reversed by setting ``archived`` to ``true`` when
     :ref:`updating <teams-update>` a team.
 
+    Admin users can delete teams. Users with org:admin and org:write permissions
+    for a team's organization can delete teams. Users with team:admin can delete
+    the team that they are admin for.
```

Comment:
Done in https://github.com/praekelt/seed-auth-api/pull/26/commits/86bf40334a84fcf9f57ef5066af54bcd896a5efe